universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -52,20 +36,28 @@ export const Onboarding = (props: OnboardingProps) => {
                   <div className="perspective">
                     <WalletSelector
                       sdk={props.sdk}
-                      onCreateClick={onCreateClick}
+                      onCreateClick={async (ensName) => {
+                        await props.walletService.createFutureWallet(ensName);
+                        history.push('/create');
+                      }}
```

Comment:
Minor thing, I think when function passed to a component is longer than one line it should be extracted

Reply:
I did it in such way just because i wanted to have history prop from RouteProps and thought that it is simplier and more readable than create function wit history arg.